src/main/scala/uk/gov/nationalarchives/tdr/api/auth/AuthorisationTags.scala

```diff
@@ -91,14 +90,14 @@ case class ValidateUserOwnsConsignment[T](argument: Argument[T]) extends Authori
   }
 }
 
-object ValidateUserOwnsFiles extends AuthorisationTag {
+case class ValidateUserOwnsFiles[T](argument: Argument[T]) extends AuthorisationTag {
```

Comment:
In this case I think you can use `Argument[UserOwnsFile]` and drop the type parameter. The type param is only necessary in `ValidateUserOwnsConsignment` because it takes a UUID _or_ a custom type.

Reply:
I wrote this before I realised we should be checking for a machine client token, so this might be a moot point.